common/lib/xmodule/xmodule/course_module.py

```diff
@@ -855,6 +855,14 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    minimum_grade_credit = Float(
```

Comment:
I understand that course staff can set this value, but it seems strange to expose an Advanced Setting to all course teams that irrelevant unless someone with "is_staff" access has enabled credit on the course.